core/store/src/main/java/org/locationtech/geowave/core/store/statistics/binning/FieldValueBinningStrategy.java

```diff
@@ -59,6 +60,11 @@ public String getDescription() {
     return "Bin the statistic by the value of one or more fields.";
   }
 
+  @Override
+  public void getFieldsUsed(final Set<String> fieldsUsed) {
```

Comment:
minor nit, but wouldn't a more appropriate name be `addFieldsUsed`   ... it just reads like a getter when its really adding in-place